libs/types-common/src/Data/Id.hs

```diff
@@ -47,14 +47,34 @@ data T
 
 data STo
 
+data Mp a
```

Comment:
Personally not a fan of these abbreviated names, even if this was already the case originally.

Reply:
Absolutely agree. I will make this clearer.